scripts/yaml-generator/src/config.lisp

```diff
@@ -413,7 +422,8 @@ common ones to include"
                    *michelson*
                    *easy-pipeline*
                    *plonk*
-                   *llvm*)
+                   *llvm*
+                   *Context*)
```

Comment:
Minor detail: why is it the only capitalized variable?

Reply:
Not any particular reason since it's called `Context` unlike some of the packages (which are mostly lower case in the package name) I capitalized the name there. CL doesn't care, so it's just a style thing